.github/workflows/build_and_test.yaml

```diff
@@ -23,6 +23,21 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - name: Run golangci-lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: v1.46.2
+          args: --build-tags=e2e
+      - name: Run yamllint
+        run: |
+          make lint-yamllint
+      - name: Run Codespell
+        uses: codespell-project/actions-codespell@master
+        with:
+          skip: .git,*.png,*.woff,*.woff2,*.eot,*.ttf,*.jpg,*.ico,*.svg,go.mod,go.sum
```

Comment:
is there any way to reuse .codespell.skip, rather than define it twice ?

Reply:
I'am afraid there is no way reusing the .codespell.skip file, shell is not supported in `with` syntax. So we can't reuse the file like `$(shell cat .codespell.skip | tr \\n ',') `

xref: [projectcontour/contour/blob/main/.github/workflows/prbuild.yaml#L38](https://github.com/projectcontour/contour/blob/main/.github/workflows/prbuild.yaml#L38)

Reply:
cool maybe a comment here explaining it would be great